examples/istio/helidon-jpa/src/main/java/io/helidon/examples/istio/GreetResource.java

```diff
@@ -0,0 +1,207 @@
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceException;
+import javax.persistence.StoredProcedureQuery;
+import javax.transaction.Transactional;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * A simple JAX-RS resource to greet you.
+ * Requires helidon-config microservice to be deployed.
+ *
+ * Examples:
+ * <p>
+ * Get default greeting message:
+ * curl -X GET http://localhost:8080/greet
+ * <p>
+ * Get greeting message for Joe:
+ * curl -X GET http://localhost:8080/greet/Joe
+ * <p>
+ * The message is returned as a JSON object.
+ */
+@Path("/greet")
+@RequestScoped
+public class GreetResource {
+
+    private static final JsonBuilderFactory JSON = Json.createBuilderFactory(Collections.emptyMap());
+
+    @PersistenceContext(unitName = "hello")
+    private EntityManager em;
+
+    @Inject
+    @RestClient
+    private GreetingProvider greetingProvider;
+
+    /**
+     * Default Constructor
+     */
+    public GreetResource() {
+    }
+
+    /**
+     * Return a worldly greeting message.
+     *
+     * @return {@link JsonObject}
+     */
+    @SuppressWarnings("checkstyle:designforextension")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public JsonObject getDefaultMessage() {
+        return createResponse("World");
+    }
+
+    /**
+     * Return a greeting message using the name that was provided.
+     *
+     * @param nick the nick to greet
+     * @return {@link JsonObject}
+     */
+    @SuppressWarnings("checkstyle:designforextension")
+    @Path("/{nick}")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Transactional
+    public Response getMessage(@PathParam("nick") String nick) {
+        Person entity = em.find(Person.class, nick);
+        JsonObjectBuilder entityBuilder = JSON.createObjectBuilder()
+                .add("nick", nick);
+        if (entity == null) {
+            JsonObject responseEntity = entityBuilder
+                    .add("error", String.format("Nick %s was not found", nick))
+                    .build();
+            return Response
+                    .status(Response.Status.CONFLICT)
+                    .entity(responseEntity)
+                    .build();
+        }
+        JsonObject responseEntity = createResponse(entity.getName());
+        return Response
+                .status(Response.Status.OK)
+                .entity(responseEntity)
+                .build();
+    }
+
+    /**
+     * Return all persons info, if available.
+     *
+     * @return {@link JsonObject}
+     */
+    @SuppressWarnings("checkstyle:designforextension")
```

Comment:
Please remove this annotation from your code. IDEs should have capability to add anything annotated with JAX-RS annotation to be added to exclusions.

Reply:
Removed it.